packages/artifact_proxy/lib/config.dart

```diff
@@ -84,6 +89,7 @@ final engineArtifactPatterns = {
   r'download.flutter.io\/io\/flutter\/arm64_v8a_release\/1.0.0-(.*)\/arm64_v8a_release-1.0.0-(.*)\.pom',
   r'download.flutter.io\/io\/flutter\/arm64_v8a_release\/1.0.0-(.*)\/arm64_v8a_release-1.0.0-(.*)\.jar\.sha1',
   r'download.flutter.io\/io\/flutter\/arm64_v8a_release\/1.0.0-(.*)\/arm64_v8a_release-1.0.0-(.*)\.jar',
+  r'download.flutter.io\/io\/flutter\/x86_64_release/\1.0.0-(.*)\/x86_64_release-1.0.0-(.*)\.pom',
```

Comment:
You'll end up with the other 3 patterns to as soon as you do an Android build I think?